use-cases.md

```diff
@@ -79,6 +79,11 @@ transited and allow for better introspection of the data.
 
 ### Event Tracing
 
+An event sent from a source may result in a sequence of additional events
+sent from various middleware devices such as event brokers and gateways.
+CloudEvents adds metadata to events to associate these events as being part
+of an event sequence for the purpose of event tracing and troubleshooting.
```

Comment:
Is this going to get a follow-on PR describing what specifically should be added to the metadata to facilitate the trace?

Reply:
there will be a follow-on PR